File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break

    for i in range(1, height + 1):

        # print spaces to the left
        for k in range(0, height - i):
            print(" ", end="")

        # print hashes for the left half
        for j in range(0, i):
            print("#", end="")

        # print the two spaces in middle
        print("  ", end="")

        # print hashes for the right half
        for l in range(0, i):
            print("#", end="")

        # print a new line
        print()


main()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    // prompt user to input height -> [1, 8]
    do
    {
        height = get_int("Height: ");
    } while (height < 1 || height > 8);

    // build the pyramid
    for (int i = 1; i <= height; i++)
    {
        // print spaces on left
        for (int k = 0; k < height - i; k++)
        {
            printf(" ");
        }
        // print hashes for the left half
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // add the middle space
        printf("  ");

        // print hashes for the right half
        for (int l = 0; l < i; l++)
        {
            printf("#");
        }

        printf("\n");
    }
}